README.md

```diff
@@ -26,8 +26,50 @@ the right to make this assignment.
 
 -----
 
-Hasjob runs on [Python][] with the [Flask][] microframework. You will need
-to install all the requirements listed in `requirements.txt` using
+### Postgres
+
+Hasjob uses Postgres >=9.4 and Redis server for development. To set up a Postgres
+DB:
+
+On OS X:
+
+    $ createuser -d hasgeek
+    $ createdb -O hasgeek -W hasjob
+
+On any Linux distribution:
+
+    $ sudo -u postgres createuser -d hasgeek
+    $ sudo -u postgres createdb -O hasgeek hasjob
+
+Edit the `/instance/settings.py` to change the variable
+`SQLALCHEMY_DATABASE_URI` to
+`postgres://hasgeek:YOUR_PASSWORD_HERE@localhost:5432/hasjob`.
+
+### Local URLs
+
+Hasjob makes use of subdomains to serve different sub-boards for jobs. To set it
+up:
+
+* Edit `/etc/hosts` file to add the entry, 
+`127.0.0.1   hasjob.dev static.hasjob.dev`
```

Comment:
OS X no longer likes multiple entries in a single line. Each name now needs to be in its own line, so:

```
127.0.0.1    hasjob.dev
127.0.0.1    static.hasjob.dev
````